tests/smokescreen/test_compartment_5.py

```diff
@@ -87,22 +88,23 @@ def test_compartment_5(page: Page, smokescreen_properties: dict) -> None:
     SubjectScreeningSummary(page).click_datasets_link()
 
     # Click on 'Show Dataset' next to the Colonoscopy Assessment
+    subject_datasets(page).click_show_datasets()
 
     # Populate Colonoscopy Assessment Details fields
 
     # ASA Grade  - I - Fit
+    subject_datasets(page).select_asa_grade_option("FIT")
```

Comment:
In order to use the Enum class AsaGradeOptions this will need to be: `AsaGradeOptions.FIT.value` instead of `"FIT"`
```suggestion
    subject_datasets(page).select_asa_grade_option(AsaGradeOptions.FIT.value)
```

Reply:
Forgot to mention that you will need to import this enum class into compartment 5